core/run/Cargo.toml

```diff
@@ -9,6 +9,9 @@ backtrace = "0.3"
 cita_trie = "3.0"
 clap = "3.1"
 futures = "0.3"
+jemalloc-ctl = { version = "0.4", package = "tikv-jemalloc-ctl", optional = true }
+jemallocator = { version = "0.4", features = ["profiling", "stats", "unprefixed_malloc_on_supported_platforms"], package = "tikv-jemallocator", optional = true }
+jemallocator-global = { version = "0.4", package = "tikv-jemallocator-global", optional = true }
```

Comment:
`jemallocator-global` introduce `cfg-if 0.1` to axon, I think this crate is not necessary